manifests/sysctl.pp

```diff
@@ -142,6 +142,11 @@
 
   # log martian packets (risky, may cause DoS)
   #net.ipv4.conf.all.log_martians = 1
+  #Lynis suggested values implemented
+  sysctl { 'net.ipv4.conf.default.log_martians' : value => '1' }
```

Comment:
@artem-sidorenko No, the closest we go is sysctl-17. This brings me back to the question:

a) Is there a way to figure out what controls from CIS has already been added into the linux-baseline? Do we maintain a excel sheet or something? Like how do make sure.. all the controls from CIS are added in linux baseline.
b) Do we add controls from the CIS Distribution Independent Linux Benchmark v1.0.1 document or any other document?